app/models/manageiq/providers/kubernetes/container_manager/event_catcher_mixin.rb

```diff
@@ -55,17 +55,23 @@ def queue_event(event)
   end
 
   def filtered?(event)
-    extract_event_data(event).nil?
+    parsed_event = extract_event_data(event)
+    parsed_event.nil? || filtered_events.include?(parsed_event[:event_type])
```

Comment:
I agree that the code should have it...I'll see about refactoring in a follow up PR.